tests/check/test_service.py

```
@@ -0,0 +1,61 @@
+import sys
```

Ruff

`sys` imported but unused

Remove unused import: `sys`

(F401)